Jenkinsfile

```diff
@@ -52,13 +109,11 @@ pipeline {
           )
         }
         failure{
-          script{
-            sh 'docker compose -f docker-compose.test.yml down --remove-orphans'
-          }
           error "Test failure. Stopping pipeline execution!"
         }
         cleanup{
           script{
+            sh 'sudo update-alternatives --set java /usr/lib/jvm/java-11-openjdk-amd64/bin/java'
```

Comment:
Why are we doing this when we're no longer setting java version to 21 explicitly anymore?
Is java 21 the default in jenkins slave now?